PosProject/src/main/webapp/WEB-INF/pages/editProduct.jsp

```diff
@@ -61,7 +61,15 @@
                 </div>
             </div>
         </div>
-
+        <div class="row">
```

Comment:
We should have a validation here. And also delete the validation from the barcode.

Reply:
Done